cms/static/sass/elements/_controls.scss

```diff
@@ -195,7 +195,7 @@
   .icon {
     display: inline-block;
     vertical-align: middle;
-    margin-right: ($baseline/4);
+    @include margin-right($baseline/4);
```

Comment:
Nitpick: Mixins/extends should be the first lines in a Sass block, so @include should be at top.